apps/rabbitci_core/web/serializers/branch_serializer.ex

```diff
@@ -0,0 +1,26 @@
+defmodule RabbitCICore.BranchSerializer do
+  use JaSerializer
+
+  require RabbitCICore.SerializerHelpers
+  alias RabbitCICore.SerializerHelpers
+
+  serialize "branches" do
+    attributes [:updated_at, :inserted_at, :name]
+    has_one :project, link: ":project_link", field: :project_id, type: "projects"
+    has_many :builds, include: RabbitCICore.BuildSerializer, link: ":builds_link"
```

Comment:
The link here should be fine as an atom `link: :project_link`.

Would being able to set the link via an anonymous function be a nice improvement? (it is not possible now)

```elixir
import RabbitCICore.Router.Helpers

  has_one :project, 
    link: &(project_path(&2, :show, &1.project_id)),
    type: "project",
    field: :project_id
```